docs/docs/guide/snippets/tutorial/counter.tsx

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import { useIdentity, useSpaces } from '@dxos/react-client';
+import React from 'react';
+
+export const Counter = () => {
+  // Get the user to log in before a space can be obtained.
+  const identity = useIdentity();
```

Comment:
Meaning you would expect it in `<App>`? I'll add a comment about it. 

There's a reason why this tutorial is building an arbitrary Shared Counter app rather than building towards one of our application templates: the focus in the tutorial is on _learning_ more than having the most cleanly-factored application.

Single component makes it much easier to follow when the form is an article (in video, it would be easier to show off editing multiple files.)